lua/telescope/builtin/files.lua

```diff
@@ -22,11 +21,21 @@ local escape_chars = function(string)
   })
 end
 
+-- Special keys:
+--  opts.search_dirs -- list of directory to search in
 files.live_grep = function(opts)
+  opts.vimgrep_arguments = opts.vimgrep_arguments or conf.vimgrep_arguments
+
   if opts.cwd then
     opts.cwd = vim.fn.expand(opts.cwd)
   end
 
+  if opts.search_dirs then
+    for i, path in ipairs(opts.search_dirs) do
+      opts.search_dirs[i] = vim.fn.expand(path)
```

Comment:
Also lets not change the dict that is passed in make a new local for that

Reply:
done